internal/cli/users_roles.go

```diff
@@ -63,3 +80,119 @@ func showUserRolesCmd(cli *cli) *cobra.Command {
 
 	return cmd
 }
+
+func addUserRolesCmd(cli *cli) *cobra.Command {
+	var inputs userRolesInput
+
+	cmd := &cobra.Command{
+		Use:     "assign",
+		Aliases: []string{"add", "associate"},
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Assign roles to a user",
+		Long:    "Assign existing roles to a user.",
+		Example: `  auth0 users roles assign <user-id>
+  auth0 users roles associate <user-id> --roles <role-id1,role-id2>
+  auth0 users roles add <user-id> -r "rol_1eKJp3jV04SiU04h,rol_2eKJp3jV04SiU04h"`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				if err := userID.Ask(cmd, &inputs.ID); err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			if len(inputs.Roles) == 0 {
+				if err := cli.pickUserRoles(&inputs); err != nil {
+					return err
+				}
+			}
+
+			var rolesToAssign []*management.Role
+			for _, roleID := range inputs.Roles {
+				rolesToAssign = append(rolesToAssign, &management.Role{
+					ID: auth0.String(roleID),
+				})
+			}
+
+			if err := ansi.Waiting(func() (err error) {
+				return cli.api.User.AssignRoles(inputs.ID, rolesToAssign)
```

Comment:
Should check to see if length of `rolesToAssign` is non-zero, otherwise will result in a 400.